tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
```

Comment:
We also tend to turn on `noImplicitThis` and `strictNullChecks` in dojo2, although these might lead to more changes... Could just raise an issue so that they are addressed in the future.